pom.xml

```diff
@@ -21,6 +21,8 @@
     <module>java-common-protos</module>
     <module>java-iam</module>
     <module>gapic-generator-java-bom</module>
+    <module>showcase</module>
```

Comment:
Adding these two modules to the root pom here may have impact to our release, that they might be released automatically. @suztomo Do you have any concerns here?

Reply:
@mpeddada1 To address the concern of accidental releases, can we define a profile so that the showcase module is part of the root project only when the profile is enabled?

Reply:
Good idea! Done.